src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,85 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
 import mate.academy.service.MovieService;
+import mate.academy.service.impl.CinemaHallService;
```

Comment:
The import statement for `CinemaHallService` suggests that it is being imported from the `impl` package. Typically, service interfaces should not be imported from implementation packages. Ensure that `CinemaHallService` is an interface and that the injector is configured to provide the correct implementation.